src/components/BuildDetailPage/BuildDetailPage.tsx

```diff
@@ -1,5 +1,325 @@
+  const { serviceContainerBuild } = useServiceContainerBuild();
+
+  const serviceContainerBuildConfigRev = useServiceContainer(buildConfigApi.getRevision);
+  const serviceContainerBuildConfigRevRunner = serviceContainerBuildConfigRev.run;
+
+  useEffect(() => {
+    if (serviceContainerBuild.data.buildConfigRevision.id && serviceContainerBuild.data.buildConfigRevision.rev) {
+      serviceContainerBuildConfigRevRunner({
+        serviceData: {
+          buildConfigId: serviceContainerBuild.data.buildConfigRevision.id,
+          buildConfigRev: serviceContainerBuild.data.buildConfigRevision.rev,
+        },
+      });
+    }
+  }, [serviceContainerBuildConfigRevRunner, serviceContainerBuild.data.buildConfigRevision]);
+
+  return (
+    <Grid hasGutter>
+      <GridItem span={12}>
+        {/* Build properties */}
+        <ContentBox padding isResponsive>
+          <Attributes>
+            <AttributesItem title={buildEntityAttributes.status.title}>
+              <BuildStatusIcon build={serviceContainerBuild.data} long />
+            </AttributesItem>
+
+            <AttributesItem title="Plain Text Log">
+              <OnceBuildIsFinished buildStatus={serviceContainerBuild.data.status}>
+                <a
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  href={`${webConfigService.getPncUrl()}/builds/${serviceContainerBuild.data.id}/logs/build`}
+                >
+                  view
+                </a>
+              </OnceBuildIsFinished>
+            </AttributesItem>
+
+            <AttributesItem title={buildEntityAttributes.id.title} tooltip={buildEntityAttributes.id.tooltip}>
+              {serviceContainerBuild.data.id}
+            </AttributesItem>
+
+            <AttributesItem title={buildEntityAttributes['user.username'].title}>
+              <Username text={serviceContainerBuild.data.user.username} length={40} />
+            </AttributesItem>
+
+            {/* Build times */}
+            <AttributesItem title={buildEntityAttributes.submitTime.title}>
+              {serviceContainerBuild.data.submitTime && createDateTime({ date: serviceContainerBuild.data.submitTime }).custom}
+            </AttributesItem>
+
+            <AttributesItem title={buildEntityAttributes.startTime.title}>
+              {(buildStatusData[serviceContainerBuild.data.status as BuildStatuses].progress !== 'FINISHED' ||
+                serviceContainerBuild.data.startTime) && (
+                <OnceBuildStarted buildStatus={serviceContainerBuild.data.status}>
+                  {serviceContainerBuild.data.startTime &&
+                    createDateTime({
+                      date: serviceContainerBuild.data.startTime,
+                      includeDateInCustom:
+                        !serviceContainerBuild.data.submitTime ||
+                        !areDatesEqual(serviceContainerBuild.data.submitTime, serviceContainerBuild.data.startTime),
```

Comment:
I think the dates on the detail page should not be shortened.
On the list, it is actually useful to condense it.